Cloudinary/Operation/01_ManageCloudinaryFiles.md

```diff
@@ -0,0 +1,182 @@
+
+1. Click the product in the product list to which you want to add an image and upload it to Cloudinary at the same time.   
+  The *Edit product* view is displayed.
+
+  ![Edit product](../../Assets/Screenshots/PIM/Products/List/EditProduct.png "[Edit product]")
+
+2. Navigate to the Cloudinary image attribute in the *Attributes* tab.
+
+  ![Cloudinary attribute](../../Assets/Screenshots/PIM/Products/List/Attributes/CloudinaryAttribute.png "[Cloudinary attribute]")
+
+3. Select the desired images in the explorer and move it to the Cloudinary image attribute box using drag and drop. When uploading a video to Cloudinary, use the corresponding Cloudinary video attribute box instead of the Cloudinary image attribute box.      
+  The images have been uploaded to the specified folder in Cloudinary. A preview tile of each image is displayed in the Cloudinary image attribute box.
+
+  ![Cloudinary images](../../Assets/Screenshots/PIM/Products/List/Attributes/CloudinaryImages.png "[Cloudinary images]")
+
+  > [Info] By default, the *actindo_temp* contextual meta data field is assigned to each image uploaded via the *PIM* module to Cloudinary. As long as a product to which the image was uploaded is not saved, the value of the *actindo_temp* field equals **1**, which means that this image is temporary. Temporary images older than 24 hours are automatically deleted from Cloudinary during the daily cronjob. When a product is saved, the value of the *actindo_temp* field of all images assigned to this Cloudinary attribute changes to **0**, which means that these images will persist in Cloudinary.
+
+  ![Actindo temp](../../Assets/Screenshots/Cloudinary/ActindoTemp.png "[Actindo temp]")
+
+4. Click the preview tile in the attribute image box to edit the image data. The image data fields below
+are displayed in the box side bar:
+  - *Tags*   
+    Enter tag names for the image. Use commas to separate the tag names. A single tag is created for each tag name.
+  - *Description (alt)*   
+    Enter a description for the image which is indicated when the image cannot be displayed. This field is automatically created for each image.
+  - *Title (caption)*   
+    Enter a title for the image. This field is automatically created for each image.
+  - *Filename*   
+    The filename is automatically adopted from the uploaded or imported image. This field is read-only. To edit a filename after uploading, open the corresponding image in Cloudinary.
+  - *Contextual meta data fields*   
+    Enter an appropriate value in the contextual meta data fields specified in the Cloudinary attribute configuration.
+  - *Structured meta data fields*   
+    Enter an appropriate value in the structured meta data fields enabled in the Cloudinary attribute configuration.
+
+    > [Info] Currently only structured meta data fields of the text and the date field type are supported by Actindo. Note to enter a date value in the appropriate date format (yyyy-MM-dd).
```

Comment:
Unsicher, ob die Formulierung "Note to enter" üblich/korrekt ist. Vielleicht:

Note that the valid date format here/in this field is ...
(Näher an unsere Standardformulierung)